docs/2026-06-25_aws_gateway_benchmark/README.md

```
@@ -0,0 +1,122 @@
+
+### Metrics captured
+
+- **Latency** — total-latency p50/p90/p95/p99, plus **TTFT** (time to first
+  token) for streaming, and throughput (RPS). Driven by the `loadgen` tool.
+- **Docker image size** — `docker image inspect` size + repo digest per gateway.
+- **Memory** — idle RSS after warmup and peak RSS under load (`docker stats`).
+- **CPU** — average CPU% under load (`docker stats`).
+
+## Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)